packages/traceloop-sdk/traceloop/sdk/client/http.py

```
@@ -127,4 +129,4 @@ def _format_error_message(self, path: str, exception: requests.exceptions.Reques
                 except Exception:
                     pass
 
-        return "\n".join(error_parts)
+        return "\n".join(error_parts)
```

Ruff (0.16.1)

[warning] No newline at end of file

Add trailing newline

(W292)